app/utils/paths.ts

```diff
@@ -16,6 +16,7 @@
  *
  */
 
+import pathLib from 'path';
```

Comment:
have you tried this with the web build, I think this was only a nodejs lib...

Reply:
moved from app.ts https://github.com/tagspaces/tagspaces/pull/1481/files#diff-ef10f37f97484a476a875416376a3e86a11d827539909c1cc4a87728d529e824L2110